docs/ref/contrib/gis/tutorial.txt

```diff
@@ -53,12 +53,9 @@ Create a Spatial Database
 
 First, create a spatial database for your project.
```

Comment:
should I remove the whole paragraph, and just take for granted that users know they have to created a db and how?